packages/common/date/src/date.ts

```diff
@@ -8,10 +8,10 @@ export type DateFnsDateType = number | Date;
 
 export const kstFormat = (date: DateFnsDateType, format: string) => _format(date, format, { locale });
 
-export const roundUpHoursInDays = (hours: number) => {
+export function roundUpHoursInDays(hours: number) {
```

Comment:
I don't think it's a big issue, but if unifying is the main thing in this PR, shouldn't `kstFormat` be changed as well?

Reply:
I agree that this is not a big issue but Wouldn't convention be important at work? That's why I posted it

I think I missed that part